pkg/minishift/clusterup/clusterup.go

```diff
@@ -168,9 +167,9 @@ func EnsureHostDirectoriesExist(host *host.Host, dirs []string) error {
 // If the requested version is < the base line version we use baseline oc to provision the requested OpenShift version.
 // If the requested OpenShift version is >= the baseline, we align the oc version with the requested OpenShift version.
 func DetermineOcVersion(requestedVersion string) string {
-	valid, _ := ValidateOpenshiftMinVersion(requestedVersion, version.GetOpenShiftVersion())
+	valid, _ := ValidateOpenshiftMinVersion(requestedVersion, constants.BackwardIncompatibleOcVersion)
 	if !valid {
-		requestedVersion = version.GetOpenShiftVersion()
+		requestedVersion = constants.MinimumOcBinaryVersion
```

Comment:
I have a hard time matching the name of these variables to what is going on in the code. Also the docs are not in sync afaikt.

Reply:
@hferentschik I have gone through the existing docs to see if any documentation needs changes and found none. But I need to add the information about the changes this PR brings in.

Reply:
I mean the godocs of the method

Reply:
Also, I am referring to the naming of the methods. Reading _DetermineOcVersion_ and its godocs I have a hard time mapping methods like _ValidateOpenshiftMinVersion_ into this. Or constants like constants.MinimumOcBinaryVersion or BackwardIncompatibleOcVersion for that matter. 

I think _ValidateOpenshiftMinVersion_ should be renamed as well as the constants. At the very least the constants should get a godoc.

Reply:
We have an issue https://github.com/minishift/minishift/issues/1261 to fix validateOpenshiftMinVersion. I will trying fix it in this issue.